src/guides/v2.3/rest/performing-searches.md

```diff
@@ -93,6 +93,29 @@ searchCriteria[filter_groups][0][filters][0][value]=2016-07-01 00:00:00&
 searchCriteria[filter_groups][0][filters][0][condition_type]=gt
 ```
 
+## Simple search using a in conditions type
+
+The following search finds all the products that are provided in the value field. Using `in` conditions type, you have to pass comma separated values to get in to the response.
```

Comment:
I'm not sure for "to get in to the response" if the in refers to the operator or just the word in? Can you reword "you have to pass comma separated values to get in to the response." - as the second half does not make clear sense in English grammar.